dts/bindings/clock/st,stm32-hsi48-clock.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022, STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32 HSI 48 fixed Clock
+  This clock is used by USB or RNG or other specific peripherals
+  It cannot be set as a sysclock source or PLL source.
+
+compatible: "st,stm32-hsi48-clock"
+
+include:
+  - name: fixed-clock.yaml
+    property-blocklist:
+      - clock-frequency
+
+properties:
+    clock-frequency:
+      type: int
+      description: |
+        Fixed 48MHz output clock frequency,
+        No need to specify it in the DTS
+      required: false
+      default: 48000000
```

Comment:
Don't use a default value. Please follow what was done for other clocks.
Then this new binding is not required any more and we can just use `fixed-clock` compatible

Reply:
using fixed_clock instead with `clock-frequency = <DT_FREQ_M(48)>;`